hardhat.config.ts

```diff
@@ -64,9 +64,19 @@ const config: HardhatUserConfig = {
       gas: 10_000_000,
       deploy: chainsDeployScriptsFolders[chainIds.mainnetFork]
     },
+    goerli: {
```

Comment:
do we really need to keep all those testnets? why not keep just one?
I'm afraid it could cost us more to maintain all of them - we already saw that each one behave differently and when making changes we will have to test all of them

Reply:
I already wrote a comment about it to ignore it for now until he arbitrum guys will get back to us.. Once they'll do that I could know which ones to keep.

A side note, you would want to have 2 for sure as testnet are not that stable (even I experienced that now when working on it..).

But for now as I mentioned above it should be ignored and the cleanup will come after we'll get answers from Arbitrum